pymc3/distributions/dist_math.py

```diff
@@ -87,6 +87,25 @@ def std_cdf(x):
     return .5 + .5 * tt.erf(x / tt.sqrt(2.))
 
 
+def normal_lcdf(mu, sigma, x):
```

Comment:
Could you remove the normal_lcdf and llcdf in https://github.com/pymc-devs/pymc3/blob/d1d2aa202803dada164cb378a4485fbba59f9e76/pymc3/examples/censored_data.py#L36-L52 ?

Reply:
done